Lib/test/test_unicodedata.py

```diff
@@ -63,33 +61,24 @@ def test_method_checksum(self):
                 (char + 'ABC').title(),
 
                 ]
-            h.update(''.join(data).encode(encoding, errors))
+            h.update(''.join(data).encode('utf-8', 'surrogatepass'))
         result = h.hexdigest()
         self.assertEqual(result, self.expectedchecksum)
 
 class UnicodeDatabaseTest(unittest.TestCase):
-
-    def setUp(self):
-        # In case unicodedata is not available, this will raise an ImportError,
-        # but the other test cases will still be run
-        import unicodedata
-        self.db = unicodedata
-
-    def tearDown(self):
-        del self.db
+    db = unicodedata
 
 class UnicodeFunctionsTest(UnicodeDatabaseTest):
 
     # Update this if the database changes. Make sure to do a full rebuild
     # (e.g. 'make distclean && make') to get the correct checksum.
-    expectedchecksum = 'c44a49ca7c5cb6441640fe174ede604b45028652'
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
+    expectedchecksum = '98d602e1f69d5c5bb8a5910c40bbbad4e18e8370'
+    @requires_resource('cpu')
```

Comment:
I would keep the `# TODO: RUSTPYTHON` and `@unittest.expectedFailure` in one commit, and remove it (if necessary) in a second commit.

Reply:
@fanninpm I just seprated it into two commits.

Reply:
@fanninpm The test didn't succeed. I would revert back to removing `@unittest.expectedFailure`.